docs/faq.rst

```diff
@@ -242,6 +242,9 @@ correct syntax. The following file names will be taken as override configuration
 
 *Issue reference:* `206`_.
 
+ keep in mind, that commands from ``postfix.cf`` are injected via system call to ``postconf -e``
+ so symbols been used should be escaped
+
```

Comment:
I'm very OK with including this into the FAQ, but has nothing to do with the topic of this PR? Perhaps belongs in a PR with the changes in `core/postfix/start.py`?